spec/lib/wipe_out/plugins/logger_spec.rb

```diff
@@ -18,17 +18,20 @@
     logged_messages = []
     allow(logger).to receive(:debug) { |log| logged_messages << log }
     user = create(:user, :with_comments)
-    comment = user.comments.first
+    first_comment = user.comments.first
+    second_comment = user.comments.second
     plan.config.logger = logger
 
     plan.execute(user)
 
-    expect(user.comments).to eq([comment])
+    expect(user.comments).to eq([first_comment, second_comment])
     expect(logged_messages).to match([
       "[WipeOut] start plan=Plan(attributes=[:last_name, :access_tokens])",
```

Comment:
Before I applied the change the updated spec was failing with 

```
[
    "[WipeOut] start plan=Plan(attributes=[:last_name, :access_tokens])",
    "[WipeOut] executing plan=Plan(attributes=[:last_name, :access_tokens]) record_class=User id=1",
    "[WipeOut] executing plan=Plan(attributes=[]) record_class=Comment id=1",
    "[WipeOut] wiped out plan=Plan(attributes=[]) record_class=Comment id=1",
    "[WipeOut] executing plan=Plan(attributes=[]) record_class=Comment id=2",
+   "[WipeOut] executing plan=Plan(attributes=[]) record_class=Comment id=2",
+   "[WipeOut] wiped out plan=Plan(attributes=[]) record_class=Comment id=2",
    "[WipeOut] wiped out plan=Plan(attributes=[]) record_class=Comment id=2",
    "[WipeOut] wiped out plan=Plan(attributes=[:last_name, :access_tokens]) record_class=User id=1",
    "[WipeOut] completed plan=Plan(attributes=[:last_name, :access_tokens])"
  ]
```

See the duplicated log lines for the second comment.

Reply:
wait, but what's the cause for that ? Isn't it an issue itself?

Reply:
@ktopolski Hey - yes this is the bug that I found reproduced in the specs - red, green, refactor (here refactore wasn't needed)

the duplicated log lines for the second comment are present because for the second comment we already have 2 sets of callbacks.

Reply:
makes sense, thank you!